CHANGES.md

```diff
@@ -1,6 +1,7 @@
 # Change Log
 
 ## Unreleased
+* Built using Avro 1.9.0
```

Comment:
Can we add a note here that default validation is now always applied, and the setting `validateDefaults` has been removed?

Reply:
Actually, I'll take care of that.  Thanks for the updates, looks great!